src/alpaka/AlpakaDataFormats/SiPixelDigiErrorsAlpaka.h

```diff
@@ -22,7 +22,6 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
       ALPAKA_ASSERT_OFFLOAD(perror_h->capacity() == static_cast<int>(maxFedWords));
 
       // TO DO: nothing really async in here for now... Pass the queue in constructor argument instead, and don't wait anymore!
-      Queue queue(device);
       alpaka::memcpy(queue, error_d, error_h, 1u);
       alpaka::wait(queue);
```

Comment:
remove the `wait` ?

Reply:
Once the `Scoped context` is implemented, the _local_ queue should be removed, and then it should be fine to remove the wait as well?